README.md

```diff
@@ -26,9 +26,9 @@
 
 ## Intro
 
-Export your excel VBA project source code for use with Git (or any VCS) from the Excel Developer ribbon.
+Export your excel VBA project source code for use with Git (or any VCS) from the Excel Developer ribbon, a pure VBA add-in for code export, no need to install a COM add-in
```

Comment:
This "punch line" statement is starting to get difficult to read and too focused on specific details like "from the Excel Developer ribbon". Perhaps we can refine this later.